### PR TITLE
New Rule: open_redirect_indeed.yml

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

File: detection-rules/open_redirect_indeed.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - "https://www.bleepingcomputer.com/news/security/evilproxy-uses-indeedcom-open-redirect-for-microsoft-365-phishing/"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    (
+      sender.email.domain.root_domain != "indeed.com"
+      and any(body.links,
+              .href_url.domain.domain == 't.indeed.com' and .href_url.path == '/r'
```
Comment (rw-access) [inline detection-rules/open_redirect_indeed.yml:13]: We have a decoder for this now

Comment (morriscode) [reply detection-rules/open_redirect_indeed.yml:13]: cc @aidenmitchell

**Comment by aidenmitchell** [comment]
/mql-mimic-exempt 245222,476900,511202

Have nothing to do with this

**Review by zoomequipd — APPROVED** [review]

**Pull request merged**